demos/drawer/permanent-drawer-above-toolbar.html

```diff
@@ -122,6 +127,17 @@
       <main class="demo-main">
         <h1 class="mdc-typography--display1">Permanent Drawer</h1>
         <p class="mdc-typography--body1">It sits to the left of this content.</p>
+
+        <div class="mdc-demo__control">
```

Comment:
Add `mdc-form-field` class, and make sure it's `@import`-ed by `demos/drawer.scss`

Ditto for other usages

Reply:
I think you mean `demo-form-field`... and maybe we should pick a class name that isn't conflatable with a component it has nothing to do with?

Reply:
Ok, so rule of thumb is if a component is followed by a <label /> it should be wrapped in mdc-form-field class?

Reply:
@kfranqueiro Yes, I believe that's right.
@moog16 Yes, we should always use `mdc-form-field` when we need to add a label next to a radio, checkbox, or switch